### PR TITLE
Upgrade deps and fix security vulnerabilities

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**

**Description of changes:**
Upgrade multiple deps to fix security vulnerabilities. Removed the terser-webpack-plugin from package.json and regenerated the package-lock.json. 

**Testing**

1. How did you test these changes?
Build and ran the smart video sending demo successfully and tested the meeting join functionality. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8010 +1,128 @@
 {
   "name": "amazon-chime-sdk-smart-video-sending-demo",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```
Comment (devalevenkatesh) [inline package-lock.json]: Due to the local file version, should we use NPM 7 and then update this?

**Review by devalevenkatesh — COMMENTED** [review]
Can you once please test the serverless deployment if working fine as well?

File: package-lock.json (hunk 1 above)
Comment (ltrung) [reply package-lock.json]: Yeah we should not convert back to lockfile version 1 :)

Comment (michhyun1) [reply package-lock.json]: Done. thanks

**Comment by michhyun1** [comment]
> Can you once please test the serverless deployment if working fine as well?

Ran it server-less and everything seems fine including console logs

**Review by ltrung — APPROVED** [review]

**Pull request merged**